units/units.cpp

```diff
@@ -1710,6 +1738,55 @@ static double getPrefixMultiplier(char p)
     }
 }
 
+/// Generate the prefix multiplier for SI units
+static double getStrictSIPrefixMultiplier(char p)
+{
+    switch (p) {
+        case 'm':
+            return 0.001;
+        case 'k':
+        case 'K':
+            return 1000.0;
+        case 'M':
+            return 1e6;
+        case 'u':
+        case '\xB5':  // latin-1 encoding "micro"
+            return 1e-6;
+        case 'd':
+            return 0.1;
+        case 'c':
+            return 0.01;
+        case 'h':
+        case 'H':
```

Comment:
Same question as for kilo, is uppercase actually SI?

Reply:
I removed support for 'K' and 'H' in the strict SI conversions